Sources/SwiftInspectorCommands/Tests/InitializerCommandSpec.swift

```diff
@@ -87,6 +87,33 @@ final class InitializerCommandSpec: QuickSpec {
         }
       }
 
+      context("when argument-name is passed") {
+        var fileURL: URL!
+
+        beforeEach {
+          fileURL = try? Temporary.makeFile(content: """
+          final class Some {
+            init(some: String, someInt: Int) {}
+          }
+          """)
+        }
+
+        it("filters out the initializers that don't have the argument names") {
```

Comment:
What happens if one of the parameter names is `_`?

Reply:
I haven't consider argument labels or `_` at all. I added this to track it https://github.com/fdiaz/SwiftInspector/issues/24 since it's out of the scope of this particular PR.

Good call though!